packages/salesforcedx-vscode-core/test/jest/conflict/timestampConflictDetector.test.ts

```diff
@@ -0,0 +1,745 @@
+import { MetadataCacheService } from '../../../src/conflict';
+import * as diffUtils from '../../../src/conflict/componentDiffer';
+import { PersistentStorageService } from './../../../src/conflict/persistentStorageService';
+import { TimestampConflictDetector } from './../../../src/conflict/timestampConflictDetector';
+
+describe('TimestampConflictDetector', () => {
```

Comment:
Also I noticed you didn't make any changes to the vscode-integration test for timestampConflictDetector.test.ts. That one seems to be more of a unit test than a uni-gration, since we're not importing vscode to test. Is that test file superseded by this one? ... can we 💣 it?!

Reply:
Yeah, it looks like that integ test file should be superseded by this one, but there is probably some additional coverage to port from that test to this unit test file.  Follow up WI?

Reply:
@randi274 I created a follow up WI to finish the conversion of all conflict-related integration tests to jest unit tests: [W-13031042](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001PO56cYAD/view).